### DISCUSSION TITLE
Is it possible to install on ZFS?

### DISCUSSION THREAD

**Discussion by lucmp** [SpiralLinux/SpiralLinux-project#285] [Q&A]
Is there any way to install Spiral on ZFS?
I suppose I can install and load the ZFS modules in live mode then "format" the disk as a ZFS pool, but will Grub find it? Will it boot? Will full disk encryption work? Does anyone know?

**Comment by geckolinux** [comment]
It might be possible but definitely not easy to install vanilla Debian on a ZFS root:
https://openzfs.github.io/openzfs-docs/Getting%20Started/Debian/Debian%20Bullseye%20Root%20on%20ZFS.html
It definitely wasn't possible at the time of the last development phase of SpiralLinux to do so with the Calamares installer.

**Reply by lucmp** [reply]
Thank you for the link. It's an interesting read.
But I have collected more information and learned that ZFS is not suitable for desktops. I have scrapped the idea.
Sorry for any trouble I may have caused.

**Reply by geckolinux** [reply]
No trouble at all. I agree that ZFS isn't the best for the desktop, at least not with a normal GRUB bootloader, since it won't have bootable snapshots. I would recommend the Btrfs defaults that SpiralLinux provides.
